jupyterhub/templates/ingress.yaml

```diff
@@ -15,23 +15,25 @@ metadata:
   {{- end }}
 spec:
   rules:
-    {{- range $host := .Values.ingress.hosts }}
-    - host: {{ $host | quote }}
-      http:
+    {{- range $host := .Values.ingress.hosts | default (list "") }}
```

Comment:
Can this default go in
https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/07cd588cf5173403c245900d180eca3b3e020491/jupyterhub/values.yaml#L504
instead? Or would that break something?

Reply:
It would have to be `[""]` if it went in the default value, and it would break if someone set it to `[]` as the logic is to force a empty-string entry to trigger at least one iteration at all time, and use the blank string entry as a sign to omit the host field.

I think I prefer it as it is because it is more robust setting `[""]` by default expose irrelevant complexity to the end user I think.

Reply:
OK!